sympy/polys/matrices/lll.py

```diff
@@ -0,0 +1,85 @@
+from math import floor as mfloor
+from typing import Tuple
+
+from sympy.polys.domains import ZZ, QQ
+from sympy.polys.matrices.exceptions import DMRankError, DMShapeError, DMValueError, DMDomainError
+
+logic_error_template = '''FAILURE IN ALGORITHM LOGIC!!!!
+Please file a bug report at https://github.com/sympy/sympy and include this input:
+{}'''
```

Comment:
Every time something like this is added to the code we end up regretting it. If there is a bug and it fails with `AssertionError` then someone will hopefully open an issue. Putting this sort of message encourages people to raise the issue without doing due diligence such as checking that they actually have the latest version installed. Once this code is in a public sympy release there will be people using this for years to come even if a newer version with bug fixes is later released.

Reply:
Ah, yeah I see your point. I didn't consider that. Ok, I'll take it out and just leave it as an assert.